.github/workflows/swagger.yml

```diff
@@ -4,35 +4,29 @@ on:
   push:
     branches:
       - master
-  pull_request:
-    branches:
-      - master
```

Comment:
shouldn't publish API specs from master.

you could change it to validate on PR if you wanted to